css-ecommerce-page/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row header-arrivals">
+      <h1 class=""> New Arrivals </h1>
+    </div>
+    <div class="row">
+      <div class="column-fifth">
+        <div class="row flex-column margin-left-right">
+          <div class="columnn-fifth width ">
```

Comment:
`width` is just setting this element to be full width, you should use `column-full` for that